src/Optimizer/Transformer/ReorderHead.php

```diff
@@ -51,9 +51,10 @@ final class ReorderHead implements Transformer
      */
     const AMP_RESOURCE_HINT_SRC_PATTERN = '#(^|[\b/])cdn\.ampproject\.org($|[\b/])#i';
 
-    /*
+    /**
```

Comment:
This is not a docblock, as it doesn't document a specific element, but rather provides an explanation for the entire group of properties. It is meant to be a standard multiline comment instead.

```suggestion
    /*
```